### ISSUE TITLE
Certificate misconfiguration

### ISSUE THREAD

**Issue by samuel-gamelin** [tobiasschlagenhaufer/cuScheduling#10]
The certificate currently being used for the [website](https://cuscheduling.net) is not valid for its www subdomain (www.cuscheduling.net).

**Comment by tobiasschlagenhaufer** [comment]
Could you give more info? What certificate?

**Comment by samuel-gamelin** [comment]
This refers to the SSL certificate installed on the [cuscheduling.net](https://cuscheduling.net) website.

In the README, the website is addressed using its www subdomain ([www.cuscheduling.net](https://www.cuscheduling.net)). Upon visiting the website on Safari on macOS, the following occurs:

<img width="690" alt="Screen Shot 2020-06-28 at 12 00 39 PM" src="https://user-images.githubusercontent.com/13603771/85952494-b9d05700-b937-11ea-8a20-aaf2b6fd31ae.png">

This does not appear to be an issue on Chrome as [www.cuscheduling.net](https://www.cuscheduling.net) is automatically redirected to https://cuscheduling.net/. However, the issue can be forced by visiting the FQDN: [www.cuscheduling.net.](https://www.cuscheduling.net.).

**Comment by tobiasschlagenhaufer** [comment]
Yes I do see that now... thanks for bringing that up. Its something I can look into, any idea why the FQDN wouldn't be configured but the common name works?

**Comment by samuel-gamelin** [comment]
From what I'm seeing, this is because the certificate does not have www.cuscheduling.net in it's list of alternative names - https://observatory.mozilla.org/analyze/cuscheduling.net#tls. Creating a new certificate with www.cuscheduling.net in the list of subject alternative names would resolve the issue. Another possibility is to replace the references to [www.cuscheduling.net](https://www.cuscheduling.net) in the README with references to [cuscheduling.net](https://cuscheduling.net) instead, so as to avoid confusion.
